frontend/src/components/layout.js

```diff
@@ -26,10 +27,8 @@ const Layout = ({ children }) => {
     <div className="bg-Neutral-100 font-raleway">
       <Header siteTitle={data.site.siteMetadata?.title || `Title`} />
       <div>
-        <main className="flex flex-col items-center bg-Shades-0 py-6">{children}</main>
-        <footer>
-          © {new Date().getFullYear()} &middot; Harvest Mission Community Church
-        </footer>
+        <main className="flex flex-col items-center bg-Shades-0">{children}</main>
+        <footer><Footer/></footer>
```

Comment:
nit: for cleanliness I think this makes more sense if the `<footer></footer>` were in the `<Footer/>` class instead of wrapping it. but not gonna block on this